### PR TITLE
Add IP throttling to Ratings and Abuse Reports

### LINKED ISSUE #16722: Add IP throttling to Ratings and Abuse Reports
We already rate-limit ratings and abuse reports per user (which falls back to IP for non-identified requests), we decided in https://bugzilla.mozilla.org/show_bug.cgi?id=1502206 to apply the same limit (1 / minute for ratings, 20 / day for abuse reports) per IP as well.

### PR DESCRIPTION
**Pull request by diox**
Fixes #16722

### CODE REVIEW

File: src/olympia/abuse/views.py [hunk 1]
```diff
@@ -9,18 +9,23 @@
 )
 from olympia.accounts.views import AccountViewSet
 from olympia.addons.views import AddonViewSet
-from olympia.api.throttling import GranularUserRateThrottle
+from olympia.api.throttling import GranularIPRateThrottle, GranularUserRateThrottle
 
 
 class AbuseThrottle(GranularUserRateThrottle):
```
Comment (bobsilverberg) [inline src/olympia/abuse/views.py:15]: Would it make sense to rename this to `AbuseUserThrottle` to be consistent with the new `AbuseIPThrottle`?

File: src/olympia/ratings/views.py [hunk 2]
```diff
@@ -78,7 +89,7 @@ class RatingViewSet(AddonChildMixin, ModelViewSet):
     ]
     reply_serializer_class = RatingSerializerReply
     flag_permission_classes = [AllowNotOwner]
-    throttle_classes = (RatingThrottle,)
+    throttle_classes = (RatingThrottle, RatingIPThrottle)
```
Comment (bobsilverberg) [inline src/olympia/ratings/views.py:92]: Similar to my comment above, should we rename `RatingThrottle` to `RatingUserThrottle`?

**Review by bobsilverberg — APPROVED** [review]
Looks good, nice work @diox. I just had a couple of questions/suggestions about naming.

r+wc

File: src/olympia/abuse/views.py (hunk 1 above)
Comment (diox) [reply src/olympia/abuse/views.py]: I agree - 93c84173fb548e4be310032a61d51d878a02c0f8

File: src/olympia/ratings/views.py (hunk 2 above)
Comment (diox) [reply src/olympia/ratings/views.py]: 93c84173fb548e4be310032a61d51d878a02c0f8

**Pull request merged**